src/locales/en/hacker/phrase.rs

```diff
@@ -0,0 +1,31 @@
+use crate::utils::seeder;
+use wasm_bindgen::prelude::*;
+use super::adjective::adjective;
+use super::ingverb::ingverb;
+use super::noun::noun;
+use super::verb::verb;
+use super::abbreviation::abbreviation;
+
+fn template(verb: &str, noun: &str, abbreviation: &str, adjective: &str, ingverb: &str) -> Vec<String> {
+    vec![
+        format!("If we {} the {}, we can get to the {} {} through the {} {} {}!", verb, noun, abbreviation, noun, adjective, abbreviation, noun),
+        format!("We need to {} the {} {} {}!", verb, adjective, abbreviation, noun),
+        format!("Try to {} the {} {}, maybe it will {} the {} {}!", verb, abbreviation, noun, verb, adjective, noun),
+        format!("You can't {} the {} without {} the {} {} {}!", verb, noun, ingverb, adjective, abbreviation, noun),

```

Comment:
For the time being we'll be skipping the ' quote character and \" in strings so can't can be cant.

Reply:
got it